File: Java/app/src/main/java/dev/ogabek/java/model/Feed.java
package dev.ogabek.java.model;

import java.util.List;

public class Feed {

    private int profilePhoto;
    private int picture;
    private String time;
    private String title;
    private String description;

    private List<Short> shorts;

    public Feed(List<Short> shorts) {
        this.shorts = shorts;
    }

    public Feed(int profilePhoto, int picture, String time, String title, String description) {
        this.profilePhoto = profilePhoto;
        this.picture = picture;
        this.time = time;
        this.title = title;
        this.description = description;
    }

    public List<Short> getShorts() {
        return shorts;
    }

    public int getProfilePhoto() {
        return profilePhoto;
    }

    public int getPicture() {
        return picture;
    }

    public String getTime() {
        return time;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }
}

File: Kotlin/app/src/main/java/dev/ogabek/kotlin/model/Feed.kt
package dev.ogabek.kotlin.model

import dev.ogabek.kotlin.model.Short

class Feed {
    var profilePhoto = 0
    var picture = 0
    lateinit var time: String
    lateinit var title: String
    lateinit var description: String

    var shorts: List<Short>? = null

    constructor(shorts: List<Short>) {
        this.shorts = shorts
    }

    constructor(profilePhoto: Int, picture: Int, time: String, title: String, description: String) {
        this.profilePhoto = profilePhoto
        this.picture = picture
        this.time = time
        this.title = title
        this.description = description
    }

}